tools/build_config/Jenkinsfile

```diff
@@ -359,6 +359,61 @@ pipeline {
       }
     }
 
+    stage('Build-Docs') {
+      steps {
+        script {
+          if (isUnix()) {
+            sh '''
```

Comment:
these code blocks (both bash and powershell) should be extracted to helper scripts in `tools/bash` and `tools/pwsh`

Ideally this should just be `module load python/3.6 && build_docs.sh` or similar.